src/mapgen/mg_schematic.h

```diff
@@ -70,6 +70,7 @@ class Server;
 #define MTSCHEM_FILE_SIGNATURE 0x4d54534d // 'MTSM'
 #define MTSCHEM_FILE_VER_HIGHEST_READ  4
 #define MTSCHEM_FILE_VER_HIGHEST_WRITE 4
+#define MTSCHEM_MAPNODE_SER_FMT_VER    28 // remember to up versions above when increasing this
```

Comment:
Unclear instructions. What's to "up"? Why? When must this be increased? Why can't we use the map serialization version for schems as well to reduce dependencies?

Reply:
@sfan5 A better idea? I took this from your older branch.

Reply:
Schematics must still be written (and read) using Zlib according to the specification.
If you were to introduce Zstd schematics (by bumping the ver to > 28), the file format of schematics needs to be bumped too.

perhaps ` // Fixed serialization version for schematics since these still need to use Zlib` would be better